website/watchPlugin.js

```diff
@@ -0,0 +1,31 @@
+const path = require('path');
+const fs = require('fs-extra');
+
+module.exports = function (context, options) {
+  const packagesDir = path.resolve(context.siteDir, '../packages');
+
+  const TSPackageFolders = [
+    path.join(packagesDir, 'docusaurus-plugin-content-docs'),
+    path.join(packagesDir, 'docusaurus-plugin-content-blog'),
+    path.join(packagesDir, 'docusaurus-plugin-content-pages'),
+  ];
+
+  TSPackageFolders.forEach((tsPackageFolder) => {
+    if (!fs.existsSync(tsPackageFolder)) {
+      throw new Error(`TS package does not exist: ${tsPackageFolder}`);
+    }
+  });
+
+  const TSPathsToWatch = TSPackageFolders.map(
+    (tsPackagePath) => `${tsPackagePath}/lib/**/*.*`,
+  );
+
+  console.log('TSPathsToWatch', TSPathsToWatch);
+
+  return {
+    name: 'docusaurus-website-watch-plugin',
+    getPathsToWatch() {
+      return [...TSPathsToWatch];
+    },
+  };
```

Comment:
Unless you confirm it's possible, I'm going to remove this, as I don't think it's possible to reload at runtime plugin code

Reply:
Nope it's not possible now. I have to stop the development server and restart it. Hmm maybe this can be an MLH project?